Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -4398,7 +4403,8 @@ ObservableList<ElementLocator> IWindowExplorer.GetElementLocators(ElementInfo El
             {
                 list.Add(new ElementLocator() { LocateBy = eLocateBy.ByName, LocateValue = name, Help = "Very Recommended (usually unique)", Active = true, IsAutoLearned = true });
             }
-            list.Add(new ElementLocator() { LocateBy = eLocateBy.ByXPath, LocateValue = ElementInfo.XPath, Help = "Recommended (sensitive to page design changes)", Active = true, IsAutoLearned = true });
+            list.Add(new ElementLocator() { LocateBy = eLocateBy.ByRelXPath, LocateValue = ((HTMLElementInfo)ElementInfo).RelXpath , Help = "Very Recommended (usually unique)", Active = true, IsAutoLearned = true });
```

Comment:
can we change "ByRelXPath" to "ByRelativeXPath" so it will be clearer to user?

Reply:
It is increasing the time to read POM elements. Someone will need to verify if its too much. In parallel i am working on handling performance issue.
Also increase time in Windows Explorer get children elements. Will just affect Web Functionality testing

There is code change needed in Selenium HTMLHelper i need to check with Ravi/Jinendra. Will update after that.